src/io.F

```diff
@@ -277,14 +277,11 @@ subroutine cem_out
            call vtk_swap_field( Ur, Ui, Ui,vtkout1)
            call vtk_swap_field( Qr(1,1),Qr(1,2),Qr(1,3),vtkout2)
            call vtk_swap_field( Qi(1,1),Qi(1,2),Qi(1,3),vtkout3)
-           call vtk_swap_field( sUr,sQr(1,1),sQi(1,1),vtkout4)
 #elif defined DRIFT
            call vtk_swap_field( cN, cP, potent,vtkout1)
 #elif defined MAXWELL
            call vtk_swap_field(EN(1,1),EN(1,2),EN(1,3),vtkout1)
            call vtk_swap_field(HN(1,1),HN(1,2),HN(1,3),vtkout2)
-           call vtk_swap_field(sEN(1,1),sEN(1,2),sEN(1,3),vtkout3)
```

Comment:
I think that deleting this should also be in the "things to clean up later" category. For now editing that line is the only way to print out e.g. currents in a full 3D simulation. (Which also reminds me--restart won't work if there are metals or 2D materials around... more things to worry about later.)